src/mixins/coordinate-grid-raster-mixin.js

```diff
@@ -636,7 +636,9 @@ export default function coordinateGridRasterMixin (_chart, _mapboxgl, browser) {
     const height = _chart.effectiveHeight()
     const margins = _chart.margins()
     const left = margins.left
+    const right = margins.right
     const top = margins.top
+    const bottom = margins.bottom
```

Comment:
Should we use a destructuring assignment (eg, `const {left, right, top, bottom} = margins`) for this? Especially with code cov breathing down our necks, it'll save a couple of lines a code. It's a minor style quibble, but I can go in and fix it if you think it's worth doing. In general, though, looks code.